src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java

```diff
@@ -110,6 +111,7 @@ public void start(Map<String, String> props) {
     maxInterval = config.getMaxInterval();
     maxRecords = config.getIterations();
     schemaFilename = config.getSchemaFilename();
+    schemaString = config.getSchemaString();
```

Comment:
Can we add some logic to make the `schema.string` and `schema.filename` configs mutually exclusive? Probably want to do that in the config class by overriding the [validate](https://kafka.apache.org/25/javadoc/org/apache/kafka/connect/connector/Connector.html#validate-java.util.Map-) method to either throw an exception or return a `ConfigInfos` object with an error in it if the user tries to specify both.